codecov.yml

```diff
@@ -0,0 +1,7 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        treshold: 0%
```

Comment:
What threshold stands for?

Reply:
As I understood it allows to monitor coverage. If next MR have coverage less it won't pass. For example if threshold = 10% and current coverage 70%, it allows to reduce coverage at next MR to 60%. I think I made a typo, I'll fix it.